### ISSUE TITLE
Implement "available" message on Client Side

### ISSUE THREAD

**Issue by stephengrice** [JHU-601/flask-production#1]
available 

Indicates the available Characters. An item omitted from the list is unavailable for selection. 
```javascript
[ 

  { 

    "name": "Colonel Mustard", 

    "color": "yellow", 

  }, 

  { 

    "name": "Miss Scarlet", 

    "color": "red", 

  }, 

  { 

    "name": "Professor Plum", 

    "color": "purple", 

  }, 

  { 

    "name": "Mr. Green", 

    "color": "green", 

  }, 

  { 

    "name": "Mrs. White", 

    "color": "white", 

  }, 

  { 

    "name": "Mrs. Peacock", 

    "color": "blue", 

  }, 

] 
```

**Comment by KenDJohnson** [comment]
@stephengrice should this be closed in favor of #15 ?

**Comment by stephengrice** [comment]
Yep, agreed! Closing
